visualization/test-requirements.txt

```diff
@@ -1,2 +1,3 @@
-dash[testing]
-webdriver_manager
+dash[testing]==2.17.0
```

Comment:
maybe we can add a new requirements folder where we put all relevant req files. Let's have "main", "inspector" and "tests" there. With tests containing what you need here + pytest and pytest-cov that we currently install in the actions pipeline

Reply:
What about datasets/requirements? Do we want to move them to the folder as well?

Reply:
Yes!